### DISCUSSION TITLE
Disabling the date picker ? [Apr 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1888] [Discord Threads]
> 🙋‍♂️ **acropolis11**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### What is the usecase for this ? Like you want to disable a column being edited  
#  
> 🙋‍♂️ **acropolis11**  
#### Because there is going to be a lot of data, it is easier to type the date in manually instead of finicking around the the date picker  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/965277000250052658)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
